app/assets/stylesheets/lisa-app.scss

```diff
@@ -52,4 +56,3 @@ p.indent {
 
 #whyGiveThisRating {
     min-height: 150px;
-}
```

Comment:
Can you re-add this closing bracket please

Reply:
Added the braces and squashed commit
Thanks